hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -113,9 +113,21 @@
     }
 
     {
-      name: hmac_test_hmac_vectors
+      name: hmac_test_hmac256_vectors
```

Comment:
Sorry, I have 2 more points to mention for this PR, as discussed together:
- I ran a regression and the timeout `+test_timeout_ns=1_000_000_000` for SHA 384/512 is too short, as a `uvm_fatal` is raised
- And also, those tests are quite long and according to my understanding the randomization scope is restricted. Indeed, if I am not wrong, only those variables are randomized:  message, wr_config_during_hash, wr_key_during_hash, key, and endian/digest/key_swap. Thus, I would suggest to reduce the `reseed` parameter to 10 for the tests with the NIST vectors. @gdessouky, @vogelpi, @andreaskurth: what do you think?

Reply:
Something else occurred to me, as `hmac_test_vectors_sha_vseq` and `hmac_test_vectors_hmac_vseq` are taking ages to run:

Do we need to run them from the `hmac_stress_all_vseq`? I think the added value is not that significant as tested in standalone and mixing it don't bring so much.

If you have a strong opinion to keep it, maybe we can reduce their weight? (to do so, I would suggest the use of a SV weighted `randcase`)

Again, what do you think: @gdessouky , @vogelpi and @andreaskurth ?

Reply:
I agree that that 1) lowering the reseed parameter makes sense as the value of these tests is limited and 2) that we should also lower the weight of these tests in the stress_all sequence. These are both good proposals.

Reply:
@martin-velay I agree that we should go ahead and exclude them from `hmac_stress_all_vseq`.  In fact, this would actually resolve this issue as well https://github.com/lowRISC/opentitan/issues/23559 with the `hmac_stress_all_with_rand_reset` failing test, or at least partially. We might also need to exclude other tests with long messages for that one as well.

Reply:
This sounds acceptable to me as well. On the other hand, increasing the wait time for outstanding accesses could be an easy fix for #23559.

Reply:
Another think I would like to say after talking with Ghada this morning. I didn't realized but also at least the key and the message and the swap are not randomized as the NIST is providing fixed test vectors and not a model. So maybe even 5 seeds per test is enough, as few space is left for the randomization.